### PR TITLE
add option pools blog post

### PR DESCRIPTION
**Pull request by morloy**

### CODE REVIEW

File: src/layouts/index.jsx [hunk 1]
```diff
@@ -144,6 +144,10 @@ const Footer = (props: LayoutProps) => (
               <Link className="nav-link" href to={`${props.prefix}/help/getting-started/`}><Trans>Getting Started</Trans></Link>
               <Link className="nav-link" href to={`${props.prefix}/help/faq/`}><Trans>FAQ</Trans></Link>
             </div>
+            <h6 className="mb-4 mt-1"><strong><Trans>Blog</Trans></strong></h6>
+            <div className="nav flex-column">
+              <Link className="nav-link" href to={`${props.prefix}/blog/pre-and-post-money-option-pools/`}>Option pools</Link>
```
Comment (bbinoy) [inline src/layouts/index.jsx]:
```suggestion
              <Link className="nav-link" href to={`${props.prefix}/blog/pre-and-post-money-option-pools/`}>Option Pools</Link>
```

File: src/layouts/index.jsx [hunk 2]
```diff
@@ -144,6 +144,10 @@ const Footer = (props: LayoutProps) => (
               <Link className="nav-link" href to={`${props.prefix}/help/getting-started/`}><Trans>Getting Started</Trans></Link>
               <Link className="nav-link" href to={`${props.prefix}/help/faq/`}><Trans>FAQ</Trans></Link>
             </div>
+            <h6 className="mb-4 mt-1"><strong><Trans>Blog</Trans></strong></h6>
```
Comment (koyanoa) [inline src/layouts/index.jsx]:
```suggestion
            <h6 className="mb-4 mt-1"><a className="nav-link" href={blogUrl} {...targetBlank}><strong><Trans>Blog</Trans></strong></a></h6>
```

**Comment by koyanoa** [comment]
@morloy I think it's a little confusing that the Blog link in the Navbar goes to the medium blog and in the footer to our landing-page Blog. One idea would be to also Link the Blog to Medium in the footer as I did in my suggestion:
image

File: src/layouts/index.jsx (hunk 2 above)
Comment (morloy) [reply src/layouts/index.jsx]: Hm, I don’t really like to turn a header into a link.

Comment (bbinoy) [reply src/layouts/index.jsx]: würde ich auch trennen...

**Pull request merged**